### PR TITLE
added global.json and nancy project.json

### PR DESCRIPTION
**Pull request by Yantrio**
## The first step to get going on coreclr + new project system

Global.json has been added to point to src,
Project.json added to nancy project,

Regarding project.lock.json files, do we want to be adding this to .gitignore or do we want to be pinning it and checking it in to source control?

### CODE REVIEW

**Comment by jchannon** [comment]
i say .gitignore but maybe @khellang has an idea

**Comment by khellang** [comment]
I closed out #2154 and #2155 because we decided to not use project.json until we're porting to .NET Core...

**Comment by Yantrio** [comment]
That day is today :) 

I can take some of the work you did on your branches to get the ball rolling though @khellang . I wasn't aware you already started this work.

File: src/Nancy/project.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            "Microsoft.CSharp": "4.0.0",
+            "System.Collections.Concurrent": "4.0.10",
+            "System.Collections.Specialized": "4.0.0",
+            "System.ComponentModel.TypeConverter": "4.0.0",
+            "System.Console": "4.0.0-beta-23516",
+            "System.Data.Common": "4.0.0",
+            "System.Diagnostics.Tools": "4.0.0",
+            "System.Dynamic.Runtime": "4.0.0-beta-23516",
+            "System.IO": "4.0.10-beta-23516",
+            "System.IO.UnmanagedMemoryStream": "4.0.0",
```
Comment (khellang) [inline src/Nancy/project.json]: What are we using this for?

File: src/Nancy/project.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            "System.Linq": "4.0.0-beta-23516",
+            "System.Net.Primitives": "4.0.0",
+            "System.Reflection": "4.0.10-beta-23516",
+            "System.Reflection.Emit": "4.0.0",
```
Comment (khellang) [inline src/Nancy/project.json]: I hope we don't use `System.Reflection.Emit` for anything?

File: src/Nancy/project.json (hunk 1 above)
Comment (Yantrio) [inline src/Nancy/project.json]: looks like we aren't, it was in jchannon's dnx branch but i can't tell why, I will revisit this and make sure all the packages are needed.

File: src/Nancy/project.json (hunk 2 above)
Comment (Yantrio) [inline src/Nancy/project.json]: maybe @jchannon can pipe up as to why he added that in his dnx branch?

Comment (jchannon) [inline src/Nancy/project.json]: No clue.

**Comment by Yantrio** [comment]
not needed anymore after #2211 and #2212

**Comment by khellang** [comment]
@Yantrio #2212 still doesn't target `dotnet5.x` though...

**Comment by Yantrio** [comment]
yeah, im going to modify the current project.json's merged in #2212 to add the new targets, that seems to be the easiest way.

**Pull request closed without merging**